documentation/book/topic-operator.adoc

```diff
@@ -1,272 +1,47 @@
 
-The operator is configured from environment variables:
+include::proc-creating-a-topic.adoc[leveloffset=+2]
 
-* `STRIMZI_CONFIGMAP_LABELS` 
-– The label selector used to identify ConfigMaps to be managed by the operator.
-  Default: `strimzi.io/kind=topic`.
-* `STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS`
-– The Zookeeper session timeout, in milliseconds. 
-For example `10000`. 
-Default: `20000` (20 seconds).
-* `STRIMZI_KAFKA_BOOTSTRAP_SERVERS`
-– The list of Kafka bootstrap servers. 
-This variable is mandatory.
-* `STRIMZI_ZOOKEEPER_CONNECT`
-– The Zookeeper connection information. 
-This variable is mandatory.
-* `STRIMZI_FULL_RECONCILIATION_INTERVAL_MS`
-– The interval between periodic reconciliations, in milliseconds.
-* `STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS`
-– The number of attempts for getting topics metadata from Kafka. 
-The time between each attempt is defined as an exponential back-off. 
-You might want to increase this value when topic creation could take more time due to its larger size (that is, many partitions/replicas). 
-Default `6`.
-* `STRIMZI_LOG_LEVEL`
-– The level for printing logging messages. 
-The value can be set to: `ERROR`, `WARNING`, `INFO`, `DEBUG` and `TRACE`. 
-Default `INFO`.
-* `STRIMZI_TLS_ENABLED`
-– For enabling the TLS support so encrypting the communication with Kafka brokers.
-Default `true`
-* `STRIMZI_TRUSTSTORE_LOCATION`
-– The path to the truststore containing certificates for enabling TLS based communication.
-This variable is mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
-* `STRIMZI_TRUSTSTORE_PASSWORD`
-– The password for accessing the truststore defined by `STRIMZI_TRUSTSTORE_LOCATION`.
-This variable is mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
-* `STRIMZI_KEYSTORE_LOCATION`
-– The path to the keystore containing private keys for enabling TLS based communication.
-This variable is mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
-* `STRIMZI_KEYSTORE_PASSWORD`
-– The password for accessing the keystore defined by `STRIMZI_KEYSTORE_LOCATION`.
-This variable is mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
+include::proc-changing-a-topic.adoc[leveloffset=+2]
 
-If the operator configuration needs to be changed the process must be killed and restarted.
-Since the operator is intended to execute within {ProductPlatformName}, this can be achieved
-by deleting the pod.
+include::proc-deleting-a-topic.adoc[leveloffset=+2]
 
+include::ref-topic-operator-environment.adoc[leveloffset=+2]
 
 === Resource limits and requests
 
 The Topic Operator can run with resource limits:
 
-* When it is deployed by the Cluster Operator these can be specified in the `resources` key of the `topic-operator-config`.
-* When it is not deployed by the Cluster Operator these can be specified on the Deployment in the usual way.
+* When it is deployed by the Cluster Operator these can be specified in the ``Kafka.spec.topicOperator.resources` property.
```

Comment:
There are two backticks at the beginning of that literal - you'll want to delete one.